ext/codesnippet.py

```diff
@@ -65,6 +65,23 @@ def depart_codesnippet_node(self, node):
 class CodeSnippet(nodes.Admonition, nodes.Element):
     pass
 
+
+# imported from sphinx/util/compat.py
```

Comment:
I guess this is sufficient, since both libraries are licensed under BSD, and the code is not significant?